clients/ui/frontend/config/webpack.dev.js

```diff
@@ -6,6 +6,9 @@ const common = require('./webpack.common.js');
 const { stylePaths } = require('./stylePaths');
 const HOST = process.env.HOST || 'localhost';
 const PORT = process.env.PORT || '9000';
+const PROXY_HOST = process.env.PROXY_HOST || 'localhost';
+const PROXY_PORT = process.env.PROXY_PORT || '4000';
+const PROXY_PROTOCOL = process.env.PROXY_PROTOCOL || 'http:';
```

Comment:
Good question, the main reason is this is a default value for local development - so we probably don't want the hassle of self signed certificates when developing against a local bff, this can be changed by setting the env var when developing remotely.